webapp/portlet/src/main/webapp/vue-apps/activity-stream/components/empty-stream/ActivityStreamEmptyMessageSpace.vue

```diff
@@ -1,40 +1,126 @@
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
 <template>
   <div class="d-flex flex-column white border-radius">
-    <img
-      :height="height"
-      :style="{maxHeight: height}"
-      src="/social-portlet/images/emptySpaceStream.png"
-      class="mx-auto"
-      loading="lazy">
-    <div class="mx-4 mb-4">
+    <div class="mx-4 my-4">
       <p v-sanitized-html="welcomeTitle"></p>
+      <v-card
+        max-width="500"
+        flat
+        class="mx-auto my-8">
+        <v-row>
+          <activity-stream-empty-message-space-icons-column
+            :icon-size="iconSize"
+            :icon-ref="userGearIcon"
+            :info-message="firstFeatureMessage" />
+          <activity-stream-empty-message-space-icons-column
+            icon-color="secondary"
+            :icon-size="iconSize"
+            :icon-ref="shareIcon"
+            :info-message="secondFeatureMessage" />
+          <activity-stream-empty-message-space-icons-column
+            :icon-size="iconSize"
+            :icon-ref="awardIcon"
+            :info-message="thirdFeatureMessage" />
+          <activity-stream-empty-message-space-icons-column
+            icon-color="secondary"
+            :icon-size="iconSize"
+            :icon-ref="trophyIcon"
+            :info-message="fourthFeatureMessage" />      
+        </v-row>
+      </v-card>
       <div v-sanitized-html="welcomeSpaceParagraph1" class="mb-4"></div>
       <div v-sanitized-html="welcomeSpaceParagraph2" class="mb-4"></div>
-      <div v-sanitized-html="conclusionParagraph"></div>
+      <div v-sanitized-html="welcomeSpaceParagraph3" class="mb-4"></div>
+      <div>
+        <ol>
+          <li v-html="welcomeSpaceNumberedParagraph1"></li>
+          <li v-sanitized-html="welcomeSpaceNumberedParagraph2"></li>
+          <li v-sanitized-html="welcomeSpaceNumberedParagraph3"></li>
+        </ol>
+      </div>
+      <div v-sanitized-html="welcomeSpaceParagraph4" class="my-4"></div>
     </div>
   </div>
 </template>
 <script>
 export default {
-  data: function() {
+  data() {
     return {
-      height: '380px',
+      emptyActivityStreamActionName: 'empty-activity-stream-spaces',
+      iconSize: '30',
+      userGearIcon: 'fa-user-cog',
+      shareIcon: 'fa-share-alt',
+      awardIcon: 'fa-award',
+      trophyIcon: 'fa-trophy',
     };
   },
+  created() {
+    document.addEventListener(this.emptyActivityStreamActionName, this.clickOnSpacesActivityStreamEmptyActionLink);
+  },
+  beforeDestroy() {
+    document.removeEventListener(this.emptyActivityStreamActionName, this.clickOnSpacesActivityStreamEmptyActionLink);
+  },
   computed: {
     welcomeTitle() {
       return this.$t && this.$t('UIActivity.label.Welcome_Space', {
         'space name': `<strong>${eXo.env.portal.spaceDisplayName}</strong>`,
       });
     },
+    firstFeatureMessage() {
```

Comment:
```suggestion
    firstFeatureMessage() {
```
please use more explicit names for variables only if really it's about just an order (first, second ...)
here it should be `emptyStreamForSpacesThatUserManage` by example. Same for others.